index.html

```diff
@@ -249,6 +249,45 @@ <h2 class="work-title-props">Multi-Post<br/>Stories</h2>
 
             </section>
 
+            <!-- About Myself -->
+            <section class="flex-box vertical flex-center transparent-bg black-fg" id="section_1" >
+
+                <h1>About Myself</h1>
+
+                <p>Hello I'm a software developer! I can help you build a product, feature or website.<br/>Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesite to contact me.</p>
+
+                <div class="flex-box vertical flex-center">
+                    <h3 class="blue-fg">LET'S CONNECT</h3>
+                    <ul class="flex-box horizontal flex-start">
+                        <li class="no-bullet">
+                            <a href="#" target="_self" rel="">
+                                <img id="social_logo_1" src="@images/social_logo_1.svg" alt="Social Logo 1"/>
+                            </a>
+                        </li>
+                        <li class="no-bullet">
+                            <a href="#" target="_self" rel="">
+                                <img id="social_logo_2" src="@images/social_logo_2.svg" alt="Social Logo 2"/>
+                            </a>
+                        </li>
+                        <li class="no-bullet">
+                            <a href="#" target="_self" rel="">
+                                <img id="social_logo_5" src="@images/social_logo_5.svg" alt="Social Logo 5"/>
```

Comment:
- [ ] It would be great if you could use the same color for the `Let's connect` heading as in the template. 
- [ ] Also your `Get my resume` button needs some extra styling so that it can match the template. Kindly address this issue.
- Your design:
![Annotation 2022-02-20 070232](https://user-images.githubusercontent.com/85113567/154831389-9cb70e4c-09ec-4514-8dec-0f368ee13905.jpg)
- Figma design:
![Annotation 2022-02-20 070251](https://user-images.githubusercontent.com/85113567/154831397-5bafdf2c-8b77-4468-965e-a86e2a7d0769.jpg)